doc/source/raysgd/raysgd_pytorch.rst

```diff
@@ -1,135 +1,124 @@
 RaySGD Pytorch
 ==============
 
-.. warning:: This is still an experimental API and is subject to change in the near future.
-
-.. tip:: Get in touch with us if you're using or considering using `RaySGD <https://forms.gle/26EMwdahdgm7Lscy9>`_!
-
 .. image:: raysgd-pytorch.svg
     :align: center
 
-The RaySGD ``PyTorchTrainer`` simplifies distributed model training for PyTorch. The ``PyTorchTrainer`` is a wrapper around ``torch.distributed.launch`` with a Python API to easily incorporate distributed training into a larger Python application, as opposed to needing to execute training outside of Python. For end to end examples, see :ref:`raysgd-pytorch-example`.
+The RaySGD ``PyTorchTrainer`` simplifies distributed model training for PyTorch. The ``PyTorchTrainer`` is a wrapper around ``torch.distributed.launch`` with a Python API to easily incorporate distributed training into a larger Python application, as opposed to needing to execute training outside of Python.
```

Comment:
What exactly does "execute training outside of python" mean? This motivation could be stronger if it was a bit more concrete (e.g., "without needing to wrap your training code in bash scripts).

Reply:
```suggestion
The RaySGD ``PyTorchTrainer`` simplifies distributed model training for PyTorch. The ``PyTorchTrainer`` is a wrapper around ``torch.distributed.launch`` with a Python API to easily incorporate distributed training into a larger Python application, as opposed to needing to wrap your training code in bash scripts.
```